src/components/GameItem.js

```diff
@@ -23,6 +27,13 @@ class GameItem extends React.Component {
           <span className={game["WL"]==="W" ? "green" : "red"}> {game["WL"]} </span>
           {score}
         </Col>
+        <Col>
+          <ScoreItem
+            score={obt.absolute.score}
+            statName={obt.absolute.stat}
+            size="medium">
```

Comment:
looking back on it, i'm not sure if a "medium" prop here is the best way to go about this, since it's pretty specific to the GameLog thing. Perhaps scratching this and putting it in the GameItem css would be better and cleaner?